packages/aws/rancher-registry.yaml

```diff
@@ -13,17 +13,11 @@ templates:
 variables:
   cni:
     - calico
-  kubernetes_version:
-    - v1.25.16+rke2r1
-    - v1.26.14+rke2r1
   registry_auth:
     - global
     - enabled
     - disabled
   docker_compose_version:
     - 2.18.1
-  rancher_version:
-    - 2.7.10
-    - 2.8.2
   cert_manager_version:
-    - 1.11.0
+    - 1.15.0
```

Comment:
does this work with earlier versions that we still  need to test like 2.8 or 2.7?